### DISCUSSION TITLE
New version of jellyfin-expo

### DISCUSSION THREAD

**Discussion by Gauvino** [jellyfin/jellyfin-meta#62] [General]
Quick question, when does the 1.6 of jellyfin-expo will come, are you waiting for the jellyfin 10.9 to release before ?

**Comment by thornbill** [comment]
In general, [we don’t do timelines](https://jellyfin.org/docs/general/faq#when-will-release-x-happen-when-will-feature-x-be-available). The main thing that blocks a release at the moment is there were some changes to how app builds are managed in expo that we need to work through. (I would actually like to move away from expo, but I lack the hardware to run a current release of macOS.) It is not dependent on 10.9. Mainly though I have just been busy with jellyfin-web lately.
